.github/workflows/python-app.yml

```diff
@@ -22,17 +22,14 @@ jobs:
     - name: Run tests
       run: make coverage
     - name: Static checks
-      if: "matrix.python-version == '3.8'"
       run: make check
     - name: Build docs with sphinx
-      if: "matrix.python-version == '3.8'"
       run: make docs
     - name: Build packages
-      if: "matrix.python-version == '3.8'"
       run: make package
     - name: Upload packages
-      if: "matrix.python-version == '3.8'"
       uses: actions/upload-artifact@v4
       with:
         name: more-itertools-packages
         path: dist/*
+        overwrite: true
```

Comment:
Testing this workaround.

References:
- https://github.com/actions/upload-artifact/issues/506
- https://devops.stackexchange.com/questions/19966/what-happens-in-github-if-artifact-names-conflicts

Reply:
Removed, maybe temporarily, so not resolving now.